test/performance/emf/parameters.yml

```diff
@@ -0,0 +1,71 @@
+receivers: ["emf"]
```

Comment:
The validator will need a generator configuration (`parameters.yml`) to know about the test case information (e.g [what metrics needs to validated or submitted to database](https://github.com/aws/amazon-cloudwatch-agent-test/pull/156/files#diff-f54f4d6d18ad5a9ec27e123541c339df116d80e9c6680528927bccc43869ddc4R17-R70), what is the [amount of run time that agent needs to run](https://github.com/aws/amazon-cloudwatch-agent-test/pull/156/files#diff-f54f4d6d18ad5a9ec27e123541c339df116d80e9c6680528927bccc43869ddc4R9)). For more information on the validator, please following [this document ](https://github.com/aws/amazon-cloudwatch-agent-test/tree/main/validator)

Reply:
Moreover, using the `receivers: ["emf"]` will generate EMF metrics to [CWA with port 25888](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/validators/basic/basic_validator.go#L35-L57)